search-improvements.rst

```diff
@@ -0,0 +1,189 @@
+.. post:: August 14, 2018
+   :tags: gsoc, search, feature
+   :author: Safwan
+   :location: DHA
+
+Improved Search
+==================================
+Have you ever struggled with a poorly documented software project?
+What about a well documented project but you can't find the right section inside the docs?
+The Read the Docs `core team`_ has realized the importance of good search for documentation
+and got me to take the challenge as a Google Summer of Code student.
+The main goal of my GSoC project was to refactor the search code together with upgrading the backend
+search engine, as well as adding more features to our search functionality like exact match search,
+case insensitive search, search as you type, suggestions and more.
+
+Google Summer of Code
+^^^^^^^^^^^^^^^^^^^^^ Contributors Wanted
```

Comment:
There's something wrong with the heading here.

![screen shot 2019-01-28 at 11 31 36 am](https://user-images.githubusercontent.com/185043/51861159-57ad9580-22f0-11e9-811f-92e2f3c8d554.png)

```suggestion
^^^^^^^^^^^^^^^^^^^^^
```